packages/shared/src/utils/contract/contract.ts

```diff
@@ -43,6 +43,10 @@ export const isVanillaContract = (contract_type: string) => /VANILLA/i.test(cont
 
 export const isCryptoContract = (underlying: string) => /^cry/.test(underlying);
 
+export const TURBOSLONG = 'turboslong';
+
+export const TURBOSSHORT = 'turbosshort';
```

Comment:
```suggestion
export const TURBOS = {
    LONG: 'turboslong';
    SHORT: 'turbosshort';
};
```
usage:
TURBOS.LONG
TURBOS.SHORT
@kate-deriv nit :)

Reply:
Noted, thank you